### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by konnov**
Added a note on IntelliJ IDEA and version bumping

### CODE REVIEW

**Comment by shonfeder** [comment]
Doh! Sorry I missed the version number update! Shall I cut a 0.7.2 to correct this bug?

**Comment by konnov** [comment]
Or shall we just republish 0.7.1 with the correct version number? I guess we don't have that many followers yet to be very strict :-)

**Comment by shonfeder** [comment]
I'd vote for doing it right. We'd have to move the tag, and that means rewriting git history on master, and that should be avoided at all costs, in principle.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -195,6 +200,7 @@ current release process is as follows:
       vl.m.n ./scripts/release-l.m.n.txt`
 - [ ] Build an updated docker container `docker build -t apalache/mv:l.m.n`
 - [ ] Push the new taged image to dockerhub `docker push apalache/mc:l.m.n`
+- [ ] Bump the version in unstable to `l.m.(n+1)` by running `mvn release:update-versions`
```
Comment (shonfeder) [inline CONTRIBUTING.md:203]:
```suggestion
- [ ] Bump the version in unstable to `l.m.(n+1)` by running `mvn release:update-versions`
- [ ] Commit the version bump to `unstable` and push
```

**Comment by konnov** [comment]
Can we introduce `0.7.1a`? :-)

**Pull request merged**